test/boost/conftest.py

```diff
@@ -0,0 +1,261 @@
+#
```

Comment:
conftest should contain only hooks/plugins/fixtures, all other classes/methods should be in pilib